src/templates/haml/get_started/index.html.haml

```diff
@@ -22,13 +22,13 @@
           = l('How to trade binaries?')
       .section-info
         = l('Follow a simple step-by-step guide to binary trading.')
-    .section
-      %a.pjaxload{:href => url_for('/get-started/beginners-faq')}
-        %div{:id => 'beginners-faq'}
+    .section.ja-hide
```

Comment:
just want to know why we changed the order i.e change Random with Begineers Faq and vice versa